scripts/generate_checksum.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHASUM_FILES="SHASUMS"
+FILE_PREFIX="eksctl-anywhere"
+
+DARWIN_TAR="darwin-amd64.tar.gz"
+LINUX_TAR="linux-amd64.tar.gz"
+
+release_manifest=$(curl --silent 'https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml')
+
+latest_release_version=$(echo $release_manifest | yq e '.spec.latestVersion' - -v)
+echo $latest_release_version
```

Comment:
I don't know where the script is run but its typically good practice to give context to echo statements like this unless they're used in some bash pipe cmd (doesn't look like it judging by the rest of the file).

```suggestion
echo "Latest release version: $latest_release_version"
```

Same applies to some of the other naked echo statements.

Reply:
Hey this was for my debugging, wont be a part of the final script.